src/core_plugins/elasticsearch/lib/clusters.js

```diff
@@ -0,0 +1,14 @@
+import Cluster from './cluster';
+
+const clusters = new Map();
```

Comment:
I [pushed a commit](https://github.com/elastic/kibana/pull/9132/commits/97ff6ceaca1bcfe35cc5333e7a2534b3a2442cef) which persists the clusters to the servers plugin object.